src/features/card/pages/card-management/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { CARD_REGISTRATION_STEPS } from '@/features/card/constants/step';
+import { useStepper } from '@/hooks/useStepper';
+import { CardInventoryPage } from '@/features/card/pages/card-inventory';
+import { AddCardPage } from '@/features/card/pages/add-card';
+import { CompleteAddPage } from '@/features/card/pages/complete-add';
+
+export const CardManagementPage = () => {
+  const { Stepper, Step, setStep } = useStepper(CARD_REGISTRATION_STEPS);
+
+  return (
+    <Stepper>
+      <Step name={CARD_REGISTRATION_STEPS[0]}>
+        <CardInventoryPage
+          onNext={() => {
+            setStep(CARD_REGISTRATION_STEPS[1]);
+          }}
```

Comment:
함수 하나를 만들어서 인자로 넘기는 형태는 어떨까요?

```suggestion
export const CardManagementPage = () => {
  const { Stepper, Step, setStep } = useStepper(CARD_REGISTRATION_STEPS);

  const handleChangeStep = (step: 0 | 1 | 2) => {
    setStep(CARD_REGISTRATION_STEPS[step]);
  }

  return (
    <Stepper>
      <Step name={CARD_REGISTRATION_STEPS[0]}>
        <CardInventoryPage
          onNext={handleChangeStep(1)}
```

Reply:
조언 감사합니다. 적용하도록 하겠습니다!